wagtail/wagtailadmin/context_processors.py

```diff
@@ -0,0 +1,7 @@
+from __future__ import absolute_import, unicode_literals
+
+from django.conf import settings
+
+
+def taggit_spaces_allowed(request):
+    return {'TAGGIT_SPACES_ALLOWED': getattr(settings, 'TAGGIT_SPACES_ALLOWED', True)}
```

Comment:
This can be replaced by a template tag (see [this file](https://github.com/wagtail/wagtail/blob/master/wagtail/wagtailadmin/templatetags/wagtailadmin_tags.py#L169-L185)).